aws/websites/metrics.pytorch.org/files/grafana.ini

```diff
@@ -0,0 +1,28 @@
+[server]
+domain = metrics.pytorch.org
+root_url = https://metrics.pytorch.org/
+
+[security]
+admin_user = {{ passwords.grafana_admin_username }}
+admin_password = {{ passwords.grafana_admin }}
+
+[users]
+allow_sign_up = false
+
+[auth.anonymous]
+enabled = true
+
+[dashboards]
+default_home_dashboard_path = /var/lib/grafana/dashboards/lambda_status.json
```

Comment:
yup you can define most things ahead of time via yaml files: https://grafana.com/docs/grafana/latest/administration/provisioning/

Grafana can hook up to a DB but if one isn't provided it defaults to a local sqlite. As long as we keep everything that matters as a provisioned resource failures shouldn't be a problem since we can just spin it up again.